src/resources/fhirReader.js

```diff
@@ -63,6 +67,21 @@ export const processBundle = (acc, resource, depth) => {
   }
 };
 
+export const getDataRange = (resourceSet, dateFormat = UI_DATE_FORMAT) => {
+  const timestamps = Object.values(resourceSet).reduce((acc, cur) => {
+    const date = cur.meta?.lastUpdated;
```

Comment:
Should we should consider `effectiveTime` , `occurrenceDateTime`, or other values, possibly nested under key paths in a resource?

(when the event occurred, rather than the time the record was recorded or updated)

Reply:
Most definitely. I've dropped in a TODO.